client.go

```diff
@@ -146,30 +180,27 @@ func (c *Client) ExecCmd(cmd *Cmd) ([]string, error) {
 		return nil, err
 	}
 
-	lines := make([]string, 0, 10)
-	for c.scanner.Scan() {
-		l := c.scanner.Text()
-		if l == "error id=0 msg=ok" {
-			if cmd.response != nil {
-				if err := DecodeResponse(lines, cmd.response); err != nil {
-					return nil, err
-				}
+	err := <-c.err
+	if err.Error() == "ok (0)" {
```

Comment:
This doesn't look like an error

Reply:
I still need to send something but instead I could check for `"error id=0 msg=ok"` before parsing the error and send `nil` to `c.err`